### PR TITLE
remove histogram, part 2

### PR DESCRIPTION
**Pull request by shintasmith**
This PR completes the removal of Histogram support from Blueflood.

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi @ChandraAddala @VinnyQ @izrik pls review

**Comment by izrik** [comment]
Looks good to me.

**Comment by ChandraAddala** [comment]
Looks good. I was hoping coverage would increase.

**Comment by shintasmith** [comment]
@ChandraAddala , yeah, isn't that weird coverage decreased??? I'll take a look at Coveralls and see what's the diff.

**Comment by shintasmith** [comment]
@ChandraAddala , it looks like the coverage decrease is due to some exception cases not covered. Let me see if I can add a  few more tests.

**Comment by usnavi** [comment]
lgtm

File: contrib/blueflood-docker/configs/blueflood.cdl [hunk 1]
```diff
@@ -200,111 +200,6 @@ CREATE TABLE IF NOT EXISTS metrics_full (
...
-  read_repair_chance=0.100000 AND
-  replicate_on_write='true' AND
-  populate_io_cache_on_flush='false' AND
-  default_time_to_live=0 AND
-  speculative_retry='NONE' AND
-  memtable_flush_period_in_ms=0 AND
-  compaction={'class': 'SizeTieredCompactionStrategy'} AND
-  compression={'chunk_length_kb': '64', 'sstable_compression': 'SnappyCompressor'};
-
 CREATE TABLE IF NOT EXISTS metrics_locator (
```
Comment (goru97) [inline contrib/blueflood-docker/configs/blueflood.cdl]: @shintasmith I was tuning Blueflood docker code and came across this change. This change can cause BF container  failure as it is using the latest released jar of BF on github, which does contain Histogram code. I would highly recommend releasing a new JAR with all these changes.

Comment (goru97) [inline contrib/blueflood-docker/configs/blueflood.cdl]: Never Mind. I released it 😃

**Pull request merged**